.github/workflows/sync-labels.yml

```diff
@@ -0,0 +1,17 @@
+name: Sync labels
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - .github/labels.yml
+jobs:
+  build:
```

Comment:
let's set the explicit permissions we want here just to be ~slightly safer -- https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs

Reply:
Good call